.github/workflows/detect-conflicting-prs.yml

```diff
@@ -0,0 +1,71 @@
+name: "Detect conflicting PRs"
+
+on:
+  workflow_dispatch:              # manually
+  # So that PRs touching the same files as the push are updated
+  push:
+  # So that the `dirtyLabel` is removed if conflicts are resolved
+  pull_request_target:            # - A pull request (even with conflicts)
+    types:
+      - synchronize               #   pushing more commits
+
+permissions:
+  # no checkouts/branching needed
+  contents: none
+  # need by "eps1lon/actions-label-merge-conflict" to manage PR label/comments
+  pull-requests: write
+
+# This allows a subsequently queued workflow run to interrupt/wait for previous runs
+concurrency:
+  group: '${{ github.workflow }}'
+  cancel-in-progress: false  # true: interrupt, false = wait for
+
+jobs:
+  detect-prs:
+    name: Detect
+    if: ${{ github.actor != 'dependabot[bot]' }} # avoid dependabot PRs
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Label conflicting PRs that are open
+        id: pr-labeler
+        uses: eps1lon/actions-label-merge-conflict@v2.0.1
+        with:
+          repoToken:  ${{ secrets.GITHUB_TOKEN }}
+          retryAfter: 30 # seconds
+          retryMax:   5  # atemps
+          dirtyLabel: conflicts
+          commentOnDirty: |
+            Oh no 😟! Conflicts have been found.
+
+            Please 🙏, take a moment and [address the merge conflicts](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts) of your pull request before we can evaluate it again.
+
+            Thanks in advance you for your effort and patience ❤️!
```

Comment:
I would like to point out a small grammatical error. It should be 'Thanks in advance for your effort and patience'. The 'you' after 'advance' and before 'for' should not be there.

Reply:
Thanks ~you~ for your eyes! 🤗